tests/e2e/vsphere_file_volume_basic_mount.go

```diff
@@ -144,6 +144,164 @@ var _ = ginkgo.Describe("[csi-file-vanilla] Verify Two Pods can read write files
 	ginkgo.It("[csi-file-vanilla] Verify Pod can read the files written by other Pod, which is deleted, when the Pod has pvc statically provisoned on same vsan file share", func() {
 		invokeTestForCreateFileVolumeAndMount(f, client, namespace, accessMode, filePath1, filePath2, true, false, true)
 	})
+	/*
+		Verify File Volume is created without specifing fstype in pv spec
+
+			1. Create StorageClass without specifying fsType
+			2. Create a PVC1 with "ReadWriteMany" using the SC from above
+			3. Wait for PVC1 to be Bound
+			4. Get the VolumeID from PV
+			5. Verify using CNS Query API if VolumeID retrieved from PV is present. Also verify
+				Name, Capacity, VolumeType, Health matches
+			6. Verify if VolumeID is created on one of the VSAN datastores from list of datacenters provided in vsphere.conf
+			7. Create Pod1 using PVC1 created above at a mount path specified in PodSpec
+			8. Create a file (file1.txt) at the mount path. Check if the creation is successful
+			9. Delete the Pod and create a PVC2 with same file share created with PVC1
+			10. Create Pod2 with PVC created above, Wait for Pod2 to be Running
+			11. Read the file (file1.txt) created in Step 8 from Pod2. Check if reading is successful
+			12. Create a new file (file2.txt) at the mount path from Pod2. Check if the creation is successful
+		Cleanup:
+			1. Delete all the Pods, pvcs and storage class and verify the deletion
+	*/
+	ginkgo.It("[csi-file-vanilla] Verify File Volume is created without specifing fstype in pv spec", func() {
+
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+		isDeletePodAfterFileCreation := true
+		staticProvisionedPVCForSecondPod := true
```

Comment:
why do we hardcode value for `staticProvisionedPVCForSecondPod` and use it in the test as below?

```
// Create Pod 2 on statically provisioned pvc
		if staticProvisionedPVCForSecondPod {
			// Creating label for PV.
```

 If this is not dynamic, can you remove using such variables?